examples/basic/package.json

```diff
@@ -1,16 +1,18 @@
 {
+  "name": "my-turborepo",
   "private": true,
   "scripts": {
-    "build": "turbo run build",
-    "dev": "turbo run dev",
-    "lint": "turbo run lint",
+    "build": "turbo build",
+    "dev": "turbo dev",
+    "lint": "turbo lint",
+    "typecheck": "turbo typecheck",
```

Comment:
We should talk about this internally. I _think_ if you use `turbo` like this here (without `npx`/`pnpx`), it refers to the global turbo binary, is that right? If that's so, I feel like we should just delete these scripts in the root package.json and recommend using `turbo` global directly? It would prevent a bunch of issues with people putting `turbo` in their package.json scripts in packages, and accidentally doing infinite recursion.

Reply:
My thought here is going to be that `create-turbo` is the entrypoint to Turborepo/monorepos for the majority of folks. Feels a little "cart before the horse" to expect someone who just heard about `turbo` 15 seconds ago from a tweet, video, etc. that was inspired enough to hit `npx create-turbo` to have global `turbo` installed. Happy to explore the squad, though, if you think it's warranted.

> I think if you use turbo like this here (without npx/pnpx), it refers to the global turbo binary, is that right?

I've been under the impression that it uses the version installed in the repo while I've been in userland. :eyes:

Reply:
My thought here is going to be that `create-turbo` is the entrypoint to Turborepo/monorepos for the majority of folks. Feels a little "cart before the horse" to expect someone who just heard about `turbo` 15 seconds ago from a tweet, video, etc. that was inspired enough to hit `npx create-turbo` to have global `turbo` installed. Happy to explore the squad, though, if you think it's warranted.

> I think if you use turbo like this here (without npx/pnpx), it refers to the global turbo binary, is that right?

I've been under the impression that it uses the version installed in the repo while I've been in userland. :eyes: